__tests__/e2e/utils/database.ts

```diff
@@ -0,0 +1,65 @@
+import mongoose, { Model } from 'mongoose'
```

Comment:
do we need this file at all? maybe just need the mongoose connection? I think the rest should work on its own without these functions...

Reply:
as discussed, can keep form collection